### DISCUSSION TITLE
xdrip+: IOB keeps dropping out of notifications area

### DISCUSSION THREAD

**Discussion by raferg** [NightscoutFoundation/xDrip#2646] [General]
As shown in the below screenshot, the IOB count (to the left of the _Carbs:21_ text) keeps dropping out of my display in xdrip+
I reset my phone and its there for a little while, but then it goes again

Does anyone know what is causing this, or if there is a setting that I am missing? 

![noIOB](https://user-images.githubusercontent.com/1743957/214220379-b527a576-4779-43d1-a384-3c59921d2147.png)

**Comment by Navid200** [comment]
This should help: https://navid200.github.io/xDrip/docs/IoB_not_showing.html

**Reply by alpacat00** [reply]
I'm using the 2024.09.17 version and still experiencing this issue (Samsung S21 with Android 14). My partner is using the same version on a different phone (Google Pixel 8) and is experiencing this as well.

**Comment by alpacat00** [comment]
Are there any plans to fix this issue?
